pkg/virt-api/webhooks/arm64.go

```diff
@@ -33,16 +33,21 @@ const (
 	defaultCPUModel = v1.CPUModeHostPassthrough
 )
 
-// ValidateVirtualMachineInstanceArm64Setting is validation function for validating-webhook
-// 1. if setting bios boot
-// 2. if use uefi secure boot
-// 3. if use host-model for cpu model
-// 4. if not use 'scsi', 'virtio' as disk bus
+// ValidateVirtualMachineInstanceArm64Setting is a validation function for validating-webhook to filter unsupported setting on Arm64
 func ValidateVirtualMachineInstanceArm64Setting(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
 	var statusCauses []metav1.StatusCause
+	validateBootOptions(field, spec, &statusCauses)
+	validateCPUModel(field, spec, &statusCauses)
+	validateDiskBus(field, spec, &statusCauses)
+	validateWatchdog(field, spec, &statusCauses)
+	validateSoundDevice(field, spec, &statusCauses)
```

Comment:
Maybe it will be more readable to change prefix from `validate` to `validateArm64*`.

Reply:
Hi @PiotrProkop , I think twice about it. As these `validate*` functions are private function and they are in the file whose name is `arm64.go` and only called by public function `ValidateVirtualMachineInstanceArm64Setting`. These hints are enough for user understand they are for Arm64 platform. Change their name from `validate` to `validateArm64*` seems redundant.

Reply:
Ok.

Reply:
Definitely not a blocker, but given that we have all functions for all architectures in one folder at the moment, the prefix would make sense. Would be a different question if we would have all arm64 validation in a seprate folder.